ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Protocol/Praos.hs

```diff
@@ -241,16 +321,22 @@ deriving instance PraosCrypto c => NoThunks (BlockInfo c)
   Protocol proper
 -------------------------------------------------------------------------------}
 
+-- | An uninhabited type for the Praos protocol.
```

Comment:
Suggestion: change `for` to `representing` or to `identifying`.

Reply:
Right, applied.